v2/dlls/Writer/template/defaultstyle.css

```
@@ -0,0 +1,65 @@
+h1, h2, h3, h4, h5, h6, p, ol, ul, pre, blockquote
+{
+	margin-top: 10px;
+	margin-bottom: 10px;
+}
+
+
+/* page layout */
+
+body
+{
+	font-family: {body-font};
```

Stylelint (17.6.0)

[error] Unknown word body-font (CssSyntaxError)

(CssSyntaxError)